Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -517,7 +532,8 @@ def audit_specs
 
       next if spec.patches.empty?
       next unless @new_formula
```

Comment:
Might be good to be able to remove the use of both `@new_formula` and `new_formula: true` being needed